pkg/skaffold/deploy/deploy_problems.go

```diff
@@ -26,49 +26,84 @@ import (
 	"github.com/GoogleContainerTools/skaffold/proto/v1"
 )
 
+const (
+	defaultMinikubeProfile = "minikube"
+)
+
+var (
+	ClusterInternalSystemErr = regexp.MustCompile(".*Internal Server Error")
```

Comment:
nit: this is exported only to be used in the `error` package. Maybe just move it there.

Reply:
It was used in `deploy` and `deploy/error` package previously and hence exported.
https://github.com/GoogleContainerTools/skaffold/pull/5491/commits/37603cc58104eac8d1b85963b4c41db8f4e74f9d#diff-a8b4a0e99c70a1b6f90aff684f68c5f25ec7650177441b92ceac2557ae3d9f35R68
and 
https://github.com/GoogleContainerTools/skaffold/pull/5491/commits/37603cc58104eac8d1b85963b4c41db8f4e74f9d#diff-8da1821d1e711320629c6b9807272ea867c0dd81e63186cfb87a9ee087b37465R66

Made private after refactor.